orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
Isn't that the idiomatic way of having enums? http://golang-basic.blogspot.com.tr/2014/07/step-by-step-guide-to-declaring-enums.html